test/integration/targets/win_disk_facts/tasks/tests.yml

```diff
@@ -0,0 +1,9 @@
+- name: get disk facts on the target
+  win_disk_facts:
+  register: disks_found
+
+- name: assert disk facts
+  assert:
+    that:
+    - disks_found.changed == false
+    #- not disks_found.ansible_facts.ansible_disk.total_disks_found == '0'
```

Comment:
Can you add a generic `- disks_found.ansible_disks.disks[0].bootable is defined` for some of the important return values. You don't have to do them all, just some.

Reply:
I have added some of the properties